public/js/modal.js

```diff
@@ -0,0 +1,19 @@
+const modalSection = document.getElementById('modal-section');
+// eslint-disable-next-line no-unused-vars
```

Comment:
is this comment `// eslint-disable-next-line no-unused-vars` necessary here?

Reply:
Yes, because im not usingi it in this file, so thats throw me an error.

![image](https://user-images.githubusercontent.com/72083744/142900337-859ffcd8-c132-46eb-94e1-2d21e2daf113.png)